catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/converter/DefaultCswRecordMap.java

```diff
@@ -51,108 +55,131 @@
         Map<String, String> localNameMap = new CaseInsensitiveMap();
 
         localNameMap.put(CswConstants.ANY_TEXT, Metacard.ANY_TEXT);
-        localNameMap.put(CswConstants.CSW_TITLE, Metacard.TITLE);
-        localNameMap.put(CswRecordMetacardType.CSW_TITLE, Metacard.TITLE);
-        localNameMap.put(CswRecordMetacardType.CSW_ALTERNATIVE, Metacard.TITLE);
+        localNameMap.put(CswConstants.CSW_TITLE, Core.TITLE);
```

Comment:
TestGmdTransformer was updated to test the new metacard mappings added here (https://github.com/codice/ddf/pull/1158/files#diff-596b92e88b80d78459215b52ded25026R84).  These other mappings are technically not new, but have been updated to use the new taxonomy constants, which are currently the same as the old ones for compatibility.  https://github.com/codice/ddf/pull/1216 updates the rest of CSW to the new taxonomy.